pyo3-ffi/src/cpython/ceval.rs

```diff
@@ -2,10 +2,22 @@ use crate::cpython::pystate::Py_tracefunc;
 use crate::object::{freefunc, PyObject};
 use std::os::raw::c_int;
 
+#[cfg(any(PyPy, Py_3_11))]
+opaque_struct!(_PyInterpreterFrame);
```

Comment:
It looks like (at least on CPython main) that this symbol resides in `cpython/pyframe.h`. Can you please create the corresponding `.rs` file and place this there?